src/jit/codegencommon.cpp

```diff
@@ -1269,18 +1269,14 @@ unsigned CodeGenInterface::InferStructOpSizeAlign(GenTree* op, unsigned* alignme
 bool CodeGen::genCreateAddrMode(GenTree*  addr,
                                 int       mode,
                                 bool      fold,
-                                regMaskTP regMask,
                                 bool*     revPtr,
                                 GenTree** rv1Ptr,
                                 GenTree** rv2Ptr,
 #if SCALED_ADDR_MODES
                                 unsigned* mulPtr,
 #endif
```

Comment:
Could you please add `// SCALED_ADDR_MODES ` to `#endif`?
`#endif // SCALED_ADDR_MODES`

These blocks are very small, but it is better to follow jit coding convention. https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#75-commenting-ifdefs